public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -903,33 +905,4 @@ describe('RuleList', () => {
       });
     });
   });
-
-  describe('Analytics', () => {
```

Comment:
why do we remove this test?

Reply:
I've removed the analytics bit from the buttons (`logInfo(LogMessages.recordingRuleFromScratch)` etc.) since we don't seem to be using those for anything.

We already track the interaction with the regular buttons (not on the CTA) through FullStory.